api/api/configuration.py

```diff
@@ -105,7 +105,7 @@ def dict_to_lowercase(mydict: Dict):
 
 def append_wazuh_prefixes(dictionary: Dict, path_fields: Dict[Any, List[Tuple[str, str]]]) -> None:
     """Append Wazuh prefix to all path fields in a dictionary.
-    
+
```

Comment:
Please try to avoid these kind of changes, it creates issues and conflicts that are unnecessary.